content/hummus.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Hummus"
+date: 2022-03-08T10:31:33-08:00
+draft: false
+serves: 6 (1/3 cup servings)
+total_time_to_make: 2 hours 15 minutes
```

Comment:
I've been writing these in minutes (135), so that one day I can format them nicely, and have an easy way to parse/sort them.

Reply:
Ugh fine 😛